packages/forklift-console-plugin/package.json

```diff
@@ -10,17 +10,19 @@
     "i18n": "i18next \"./src/**/*.{js,jsx,ts,tsx}\" [-oc] -c ./i18next-parser.config.mjs",
     "build": "NODE_ENV=production webpack",
     "start": "webpack serve",
-    "lint": "eslint . && stylelint \"packages/**/*.css\" --allow-empty-input",
-    "lint:fix": "eslint . --fix && stylelint \"packages/**/*.css\" --allow-empty-input --fix",
+    "lint": "eslint . && stylelint \"src/**/*.css\" --allow-empty-input",
+    "lint:fix": "eslint . --fix && stylelint \"src/**/*.css\" --allow-empty-input --fix",
     "test": "TZ=UTC jest",
     "test:coverage": "TZ=UTC jest --coverage",
     "test:updateSnapshot": "TZ=UTC jest --updateSnapshot"
   },
-  "devDependencies": {
+  "dependencies": {
     "@kubev2v/webpack": "*",
     "@kubev2v/legacy": "*",
     "@kubev2v/common": "*",
-    "@migtools/lib-ui": "^8.4.1",
+    "@migtools/lib-ui": "^8.4.1"
+  },
```

Comment:
Some more things should be in regular dependencies (i.e. used in browser vs used for building)?
  - @openshift-console/dynamic-plugin-sdk
  - @patternfly/react-core
  - @patternfly/react-table
  - react
  - react-i18next
  - react-router
  - react-router-dom

Reply:
:+1: , will move them to peer (on run time, they come from console)